### ISSUE TITLE
css data urls confuse SingleFile

### ISSUE THREAD

**Issue by jrmuizel** [gildas-lormeau/SingleFile#36]
The following [page](https://people-mozilla.org/~jmuizelaar/tmp/test.html) can't be saved with SingleFile:
```html
<link type="text/css" rel="stylesheet" href="data:text/css; charset=utf-8,div {width:100%}">
<div>Food</div>
```
An exception is thrown in decodeDataURI when trying to decode "div {width:100%}"

**Comment by jrmuizel** [comment]
Pull request #37 should fix this

**Comment by gildas-lormeau** [comment]
Thanks for the issue and the pull request! I did not have time to work on the extension when you submitted it so I didn't reply. I'm sorry for that. I rewrote recently most of the extension because the "Single Core" extension has been removed from the store. Finally, I use `window.fetch` instead of a `XHR`. 
The issue is fixed in SingleFile version 1.0.15.
